torch/_inductor/ir.py

```diff
@@ -5186,6 +5186,11 @@ def codegen_kwargs(self, skip_out=False):  # type: ignore[no-untyped-def]
 
     def codegen_size_asserts(self, wrapper) -> None:  # type: ignore[no-untyped-def]
         if config.size_asserts and not V.graph.cpp_wrapper:
+            # if there is some symbol in size or stride, we couldn't generate asserts
+            if not isinstance(sympy_product(self.get_size()), int):
+                return
+            if not isinstance(sympy_product(self.get_stride()), int):
+                return
```

Comment:
I think you want to be checking `isinstance(..., (int, sympy.Integer))`

Reply:
You can ping me to approve CI.

Reply:
@jansel I though this PR should fix these test failed now, may I ask you to run a round of CI test for it, and PTAL, thanks!